docs/Module3/5_ELN_using_built-in_schema/M3_2_1_creating_entries_built-in_schema.md

```diff
@@ -1,29 +1,29 @@
-# Creating NOMAD entries using the ELN built-in ELN schemas
-In this section, you will learn how to use NOMAD's built-in ELN schemas to create entries of entities and activities in your experiment.
+# Creating NOMAD Entries Using the ELN Built-in ELN Schemas
+In this section, you will learn how to use NOMAD's built-in ELN schemas to create entries for entities and activities in your experiment.
 
-After planning for your experiments, you would start by documenting your experimental work. This involves creating a record of all the materials and instruments that will be used during your experiment, or **entities**. In addition, the conditions under which the materials will be processed and the methods of measurement will be outlined, i.e., **activities**
+After planning for your experiments, you would start by documenting your experimental work. This involves creating a record of all the materials and instruments that will be used during your experiment, i.e., **entities**. In addition, you should outline the conditions under which the materials will be processed and the methods of measurement, i.e., **activities**.
 
-This record in NOMAD is made by creating entries for each entity and activity that will be used in your experiments.
+In NOMAD, this is done by creating entries for each entity and activity that will be used in your experiments.
 
 ![Alt text](../images/overview/3.png)
 
 
 
-### **Step 1:** Create a new upload for the electronic lab note book for the experiments. This upload will be your ELN of the planned experiment. 
```

Comment:
Lets remove **Step 1** , and use the steps number in the images to refer to individual steps. What do you think?